db/schema.rb

```diff
@@ -0,0 +1,51 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2022_12_02_101257) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "bookings", force: :cascade do |t|
+    t.date "date"
+    t.text "time"
+    t.string "city"
+    t.text "disease_desc"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "user_id", null: false
+    t.bigint "doctor_id", null: false
+    t.index ["doctor_id"], name: "index_bookings_on_doctor_id"
+    t.index ["user_id"], name: "index_bookings_on_user_id"
+  end
+
+  create_table "doctors", force: :cascade do |t|
+    t.string "first_name"
+    t.string "last_name"
+    t.string "specialization"
+    t.integer "phone"
+    t.string "email"
+    t.string "city"
+    t.text "bio"
+    t.text "photo"
```

Comment:
- I understand to add a photo inside the DB is a way to solve this, but what about if in the future you want to export this DB to another server? This could make a lot of diff problems. I consider the best way to use photos is to add a photos folder on the project, add a unique name, and create a table to organize this for users/others (just a comment). 🐱

Reply:
Done, thanks